src/extrinsic/mod.rs

```diff
@@ -41,6 +44,9 @@ use crate::{
     Error,
 };
 
+/// A reasonable default for `era_period`
```

Comment:
Bringing offline discussion here - I am not using any science to determine this is reasonable, I saw it in `txwrapper`, it is power of 2, and subjectively it seems like an amount of time where the Tx could sit in a pool for a little if needs to but it would be hard to create a replay attack within that time period. Certainly open to ideas for how to come up with a better number (or just better rationale for this number).

There also is no default in the metadata.